config/config.go

```diff
@@ -640,14 +640,15 @@ type Performance struct {
 	ProjectionPushDown bool   `toml:"projection-push-down" json:"projection-push-down"`
 	MaxTxnTTL          uint64 `toml:"max-txn-ttl" json:"max-txn-ttl"`
 	// Deprecated
```

Comment:
It seems the Deprecated variables are defined bellow here. Besides, why we use the config rather than the system variables?

Reply:
I think it only indicates `MemProfileInterval` is deprecated. We use config is due to `AnalyzePartitionConcurrencyQuota` is used to pre-create the session pool in domain when tidb-server started.